autoload/airline/extensions.vim

```diff
@@ -156,10 +156,10 @@ function! airline#extensions#load()
     call add(s:loaded_ext, 'netrw')
   endif
 
-  if has("terminal") || has('nvim')
-    call airline#extensions#term#init(s:ext)
-    call add(s:loaded_ext, 'term')
-  endif
+  " if has("terminal") || has('nvim')
+    " call airline#extensions#term#init(s:ext)
+    " call add(s:loaded_ext, 'term')
+  " endif
```

Comment:
why is this disabled?

Reply:
I'm sorry. The reason is the same as below.